python/tests/stress/arcticdb/version_store/test_stress_multicolumn.py

```diff
@@ -38,7 +39,18 @@ def make_periods(start_date, end_date, freq, range_type="b"):
     return [r for r in ranges if len(r) > 0]
 
 
-@pytest.mark.parametrize("lib_type", ["lmdb_version_store_big_map", "s3_version_store", "s3_version_store"])
```

Comment:
Please note the original duplicated `s3_version_store` is just a mistake when the `mongo_version_store` was mechanically replaced for the external release.

Since you're touching this line, you should fix that by removing the duplication.

Note: I am reviewing from the last file up and there are many similar examples above.